addons/characters/CfgVehicles.hpp

```diff
@@ -0,0 +1,69 @@
+#define UNLOCK_MALARIA_INFECTED_CIVILIAN  \
+    editorSubcategory = "EdSubcat_Personnel_MalariaInfected"; \
+    scope = 2; \
+    scopeCurator = 2; \
+    class EventHandlers { \
+        init = "(_this select 0) setIdentity selectRandom [ \
+            'BIS_Ambient01_sick', \
+            'BIS_Ambient02_sick', \
+            'BIS_Ambient03_sick', \
+            'BIS_Arthur_sick', \
+            'BIS_Howard_sick', \
+            'BIS_John_sick', \
+            'BIS_Lucas_sick', \
+            'BIS_Renly_sick' \
+        ]; \
+        (_this select 0) setDamage 0.45"; \
```

Comment:
`setDammage` has global effects, every time someone connects to server this will run

Reply:
It also overwrites XEH for these classes, somewhat ironically.

Reply:
Also, while `setIdentity` is a local effects command, `selectRandom` will lead to every client picking a different, not synchronized identity :D

Reply:
Just me messing around, but a global init script could work if seeds are used for randomization, e.g. the second part of the object net id:
```sqf
params ['_unit'];

if (isServer) then {
    _unit setDamage 0.45;
};

private _identities = ['BIS_Ambient01_sick', 'BIS_Ambient02_sick', 'BIS_Ambient03_sick', 'BIS_Arthur_sick', 'BIS_Howard_sick', 'BIS_John_sick', 'BIS_Lucas_sick', 'BIS_Renly_sick'];
private _identity = _identities select floor (parseNumber (_unit call BIS_fnc_netId splitString ":" param [1, '0']) random count _identities);
_unit setIdentity _identity;
```

I do not know if `setIdentity` even works in init. Yesterday I helped a poor soul on reddit with setSpeaker and it wouldn't work even in InitPost until a 5 second delay was added.

Reply:
I'll give another thought about selecting the identity.
`setIdentity` most certainly work in the init. Miller's identity is set that way.

Reply:
It's a campaign unit though. Tried Miller in MP?

Reply:
Funny enough Miller works fine in MP even though `setIdentity` in the init line is not postponed by a frame. Dunno how that works...

Reply:
obsolete